### ISSUE TITLE
Cassandra timestamp with miliseconds 

### ISSUE THREAD

**Issue by sergiormb** [TablePlus/TablePlus#1976]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Amazon Keyspaces (Apache Cassandra)

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
TablePlus 3.6.1 (320) in Mac

3. The steps to reproduce this issue:
- Create a register timestamp with miliseconds
- Now you can see date in TablePlus without miliseconds
Example:
2020-06-29 08:59:32 +0000

- And if you check the database, you can see 2020-06-29 08:59:32.107000

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by k0mrade** [comment]
I see the same behavior.
TablePlus shows 2021-03-17 08:18:46 +0000
cqlsh shows 2021-03-17 08:18:46.816000+0000
Please add a possibility to change a datetime formatting pattern

**Comment by cloudhx** [comment]
The same issue here, this causes unique key violation when moving data from Cassandra to other db where timestamp is the key.

**Comment by lxwzy** [comment]
Some issue.
cqlsh show like '2021-07-06 07:36:19.774000+0000', TablePlus shows '2021-07-06 07:36:19 +0000'.
If I set '2021-07-06 07:36:19.774000+0000' in TablePlus, it shows error `marshaling error: unable to parse date '2021-07-06 07:36:19.774000+0000': marshaling error: Milliseconds length exceeds expected (6)`.
If I set "2021-07-06 07:36:19 +0000" in TablePlus, it shows `marshaling error: unable to parse date '2021-07-06 07:36:19 +0000': marshaling error: Unable to parse timezone ' +0000'`
